Public/Src/Cache/ContentStore/Library/Utils/BandwidthChecker.cs

```diff
@@ -60,7 +71,8 @@ async Task impl()
             {
                 // This method should not fail with exceptions because the resulting task may be left unobserved causing an application to crash
                 // (given that the app is configured to fail on unobserved task exceptions).
-                var minimumSpeedInMbPerSec = _bandwidthLimitSource.GetMinimumSpeedInMbPerSec();
+                var minimumSpeedInMbPerSec = _bandwidthLimitSource.GetMinimumSpeedInMbPerSec() * _config.BandwidthLimitMultiplier;
+                minimumSpeedInMbPerSec = Math.Min(minimumSpeedInMbPerSec, _config.MaxBandwidthLimit);

```

Comment:
I'm not sure what is the reason for having MaxBandwidthLimit. Does it make sense to set it in any possible scenarios?

Reply:
It is so that we don't get an over-optimistic estimate of what the bandwidth should be. If for some reason we have a bunch of copies that are faster than normal, we could end up in a state where the expectation would be unrealistic.